### ISSUE TITLE
Can't boot PC with Arc A770 16 GB LE and XMP on X570 Aorus Elite

### ISSUE THREAD

**Issue by noc-tae** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#166]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

N/A

### Processor / Processor Number [Required]

Ryzen 7 3700X

### Graphic Card [Required]

Intel Arc A770 16GB LE

### GPU Driver Version [Required]

- [ ] 31.0.101.3491 (*latest*)
- [X] 31.0.101.3490
- [ ] 31.0.101.3430
- [ ] 31.0.101.3276
- [ ] 30.0.101.3268
- [ ] 30.0.101.3259
- [ ] 30.0.101.1743

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H1 (*October 2021 Update*)
- [ ] Windows 10 21H2 (*November 2021 Update*)
- [ ] Windows 10 21H1 (*May 2021 Update*)
- [ ] Windows 10 20H2 (*October 2020 Update*)
- [ ] Other (Please specify)

### Intel System Support Utility report

[intel-ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/9882323/intel-ssu.txt)

### Description and steps to reproduce [Required]

Since I installed my Intel Arc card, I cannot boot my PC while using XMP.

This problem is not limited to the latest BIOS version, but applies to all BIOS versions which support ReBAR.

I don't know how to reproduce this—it just triggers for me.

This is not due to the rest of my setup, if I use another card (AMD Radeon 5700XT, it works smoothly after re-enabling XMP in the BIOS)

### Device / Platform

AM4 / X570 Aorus Elite

**Comment by AntVNS** [comment]
You are on AM4 platform. I suppose when you say ReBar you are mean SAM? 
Have you tired to disable XMP and set the DDR settings manually. Does it boot with the ARC?

**Comment by noc-tae** [comment]
Well, yes, but isn't SAM just AMD's marketing name for ReBar? I have, but it only boots with stock RAM timings. 

I'm fairly sure I figured out what's causing it though: booting Ubuntu 22.04 with Intel's kernel for Arc Graphics support. I tried booting a different kernel and it works with XMP. Going back to Intel's Ubuntu kernel breaks XMP again and also messes up the graphics driver. Guess I'll figure out where to post my new issue. ¯\_(ツ)_/¯
